config/serverconfigupdater-common.toml

```diff
@@ -10,7 +10,7 @@
 	#By default Folders are only deleted if they are empty. Set to true to change that.
 	deleteFoldersWithContent = false
 	#This is intended for deleting datapacks and/or craft tweaker scripts. The file will be deleted every launch if it exists! No access to saves or world folder. Specify the path to the file. Comma Separated List. Example: scripts/badscript.zs
-	filesToDelete = "kubejs/assets/mythicbotany/lang/en_us.json,kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/occultism/spirit_fire.js,kubejs/server_scripts/enigmatica/kubejs/base/tags/items/forge/tools.js,kubejs/data/tetra/loot_tables/actions/geode.json,kubejs/assets/enigmatica/tips/paraglider.json,kubejs/data/minecraft/loot_tables/chests/end_city_treasure.json,config/configswapper/expert/randompatches.txt,config/configswapper/normal/randompatches.txt,config/configswapper/normal/occultism.txt,config/configswapper/normal/immersiveengineering.txt,config/configswapper/normal/astralsorcery.txt,config/configswapper/expert/occultism.txt,config/configswapper/expert/immersiveengineering.txt,config/configswapper/expert/astralsorcery.txt"
+	filesToDelete = "defaultconfigs/immersiveengineering-server.toml,kubejs/assets/mythicbotany/lang/en_us.json,kubejs/server_scripts/enigmatica/kubejs/base/recipetypes/occultism/spirit_fire.js,kubejs/server_scripts/enigmatica/kubejs/base/tags/items/forge/tools.js,kubejs/data/tetra/loot_tables/actions/geode.json,kubejs/assets/enigmatica/tips/paraglider.json,kubejs/data/minecraft/loot_tables/chests/end_city_treasure.json,config/configswapper/expert/randompatches.txt,config/configswapper/normal/randompatches.txt,config/configswapper/normal/occultism.txt,config/configswapper/normal/immersiveengineering.txt,config/configswapper/normal/astralsorcery.txt,config/configswapper/expert/occultism.txt,config/configswapper/expert/immersiveengineering.txt,config/configswapper/expert/astralsorcery.txt"

```

Comment:
why this?

Reply:
All of that's moved to the config swapper configs, so it's to avoid having the normal values reset the expert ones after an update

Reply:
On second thought, I guess it's not actually needed

Reply:
removed